mycroft/client/speech/__main__.py

```diff
@@ -171,18 +171,29 @@ def handle_open():
     EnclosureAPI(bus).reset()
 
 
-def main():
-    global bus
-    global loop
-    global config
-    reset_sigint_handler()
-    PIDLock("voice")
-    bus = MessageBusClient()  # Mycroft messagebus, see mycroft.messagebus
-    Configuration.set_config_update_handlers(bus)
-    config = Configuration.get()
-
-    # Register handlers on internal RecognizerLoop bus
-    loop = RecognizerLoop()
+def is_alive(message=None):
+    """Respond to is_alive status request."""
+    status = alive_status.is_set()
+    if message:
+        bus.emit(message.response({'status': status}))
+    return status
+
+
+def on_ready():
+    alive_status.set()
```

Comment:
Since these can be overridden the setting in the alive_status here may not be the best way to do this.

Reply:
ah good point, maybe I'll shift them to where the hooks are called from.

Does the overall approach make sense though? Should we be moving straight to an external monitoring service or something else?

Reply:
It sort of makes sense with the other things the enclosure does. I don't have a better suggestion...